.github/workflows/ci.yml

```
@@ -86,6 +86,18 @@ jobs:
         with:
           fetch-depth: 2
 
+      # Temporary fix to unblock CI
+      # We revert to an older Windows SDK for 32-bit Windows MSVC builds.
+      # See issue https://github.com/rust-lang/rust/issues/137733 for more details.
+      - shell: powershell
+        if: ${{ matrix.name == 'i686-msvc-1' || matrix.name == 'i686-msvc-2' || matrix.name == 'dist-i686-msvc' }}
+        run: |
+          $vs = &"${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe" -latest -format value -property installationPath
+          cmd /c "`"$vs\VC\Auxiliary\Build\vcvarsall.bat`" x86 10.0.22621.0 & SET > envs.txt"
```

Oh I'd guess you're getting some banner text from vcvarsall.bat. Trying redirecting it to `NULL`.
```suggestion
          cmd /c "`"$vs\VC\Auxiliary\Build\vcvarsall.bat`" x86 10.0.22621.0 > NULL & SET > envs.txt"
```

---

Yeah I fixed that already, thanks :) But it seems like the build fails with the environment variables configured :/